### ISSUE TITLE
Conjur Secrets Plugin Version1.0.12 in our Cloudbees jenkins 2.332.4.1 causes null pointer exception with credentials drop down everywhere

### ISSUE THREAD

**Issue by ujjwal2190** [jenkinsci/conjur-credentials-plugin#29]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
  Paste the output here
  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

We use Cloudbees jenkins on modern platforms (AWS EKS) with operatioon center and managed controllers. Version is 2.332.4.1.
We have a critical organisational requirement to use cyberark for centralized secure credential storage and so we are working with cyberark to have conjur plugin installed in jenkins for the integration so we can run pipeline jobs that uses cyberark conjur secret manager to retrieve credentials for jenkins pipelines instead of directly using jenkins credentials (usual way). Installing the plugin and configuring it in jenkins system tab causes the "A problem occurred while processing the request.
Logging ID=3a71dcf8-7067-4d61-b5cb-b513d9893f8e" similar errors within jenkins everywhere the credential dropdown options exist.  Separate logging IDs for each credential dropdown. This also impacts managed files like npmrc etc. Even regular and normal pipeline builds fail immediately with java.lang.nullpointer exception when builds are run even before Jenkinsfile can be fetched from GitHub SCM.  Upon looking at jenkins logs with these logging IDs it points to null pointer exception stack trace  relating to conjur.
For now we have disabled the plugin so jenkins is not impacted for our end users but we also have urgent requirement to have conjur set up in jenkins without impacting anything else.

### Reproduction steps

1. Install conjur Secrets Plugin Version 1.0.12 in any controller in  cloudbees jenkins 2.332.4.1
2. Configure it within jenkins controller system tab

### Expected Results

1. Expected result is to have the plugin work as usual without impacting jenkins in any way

### Actual Results

1. A problem occurred while processing the request with logging ID error for just about any credential dropdown within jenkins causing null pointer exception in builds. Even if we ignore the "A problem occurred while processing the request with logging ID error" in jenkins controller system tab or managed files wherevere credential dropdown exists, just the builds alone fail most of the times right away with java.lang.nullpointer exception and upon digging further into the logs it points to conjur secrets plugin.

### Anything else?

Its very much related to https://github.com/jenkinsci/conjur-credentials-plugin/issues/16
But since our version we installed is 1.0.12 (which is what is available for our jenkins that is on version 2.332.4.2) , we still see this error.
![cred-dropdown-error-everywhere](https://github.com/jenkinsci/conjur-credentials-plugin/assets/49960666/f5abc926-e153-40c5-a050-53b3defd1b8e)

**Comment by thangsks** [comment]
Yes, i also faced same issue. uninstalled and restarted jenkins to fix this issue

**Comment by ujjwal2190** [comment]
Thanks for letting me know. We have it currently uninstalled and those errors are gone. However, we have a need to use this plugin so trying to figure out how to make it work for our jenkins at version 2.332.4.2

**Comment by venkatesh588** [comment]
@ujjwal2190 You are not able to execute all the pipelines once this nullpointer issue comes?

**Comment by ujjwal2190** [comment]
It impacts all such pipelines that rely on credentials from jenkins controller to run the jobs.

**Comment by ujjwal2190** [comment]
We saw it for both when we ran pipelines. Even pipelines not tied to conjur would throw that error somehow so we worked with CyberArk for some patch but that too didn't yield results. We were not in a position to upgrade jenkins overall to be able to use the newer version of the plugin that is claimed to work with the upgraded version of jenkins. But as per compatibility matrix chart,  when this issue was created it should work. If this version of plugin with our version of jenkins (which is compatible as per the docs and usage) doesn't work then we can try for overall upgrade as our last best bet.

**Comment by ujjwal2190** [comment]
That is true for me too.
